code/__HELPERS/unsorted.dm

```diff
@@ -1307,10 +1307,10 @@ proc/pick_closest_path(value)
 
 	if(!istype(C))
 		return
-
+	var/old_color = C.color
 	C.color = flash_color
 	spawn(0)
-		animate(C, color = initial(C.color), time = flash_time)
+		animate(C, color = old_color, time = flash_time)
```

Comment:
why is this in a spawn, i'm not seeing a sleep.

animate doesn't block.

This spawn seems unneeded.